pgx-utils/src/rewriter.rs

```diff
@@ -566,8 +537,18 @@ impl PgGuardRewriter {
                         if suffix_arg_name && ident.ident.to_string() != "fcinfo" {
                             let ident = Ident::new(&format!("{}_", ident.ident), ident.span());
                             arg_list.extend(quote! { #ident, });
+                            if type_matches(ty.ty.as_ref(), "()")
+                                || extract_type_from_option(ty.ty.as_ref()).is_some()
+                                || type_matches(&ty.ty, "pg_sys :: FunctionCallInfo")
+                                || type_matches(&ty.ty, "pgx :: pg_sys :: FunctionCallInfo")
```

Comment:
Why the spaces in `pgx :: pg_sys :: FunctionCallInfo`?

Reply:
I think that's due to how [`format!("{}", quote!`](https://github.com/tcdi/pgx/blob/f2c54dc38e6b28cb7dcf43ef9f84df36e7108278/pgx-utils/src/rewriter.rs#L750) works. I was just matching the style seen elsewhere.